test/e2e/examples_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"testing"
+
+	agonesv1 "agones.dev/agones/pkg/apis/agones/v1"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestXonoticGameServerReady(t *testing.T) {
+	t.Parallel()
+	namespace := "agones-system"
+	if namespace == "" {
+		t.Fatal("gameserversNamespace must be set")
+	}
+	t.Logf("Using namespace: %s", namespace)
+
+	gs := framework.DefaultGameServer(framework.Namespace)
```

Comment:
You could probably just start with a blank GameServer? You don't need the default one.